portal-web/test/functional/com/liferay/portalweb/paths/portlet/documentsandmedia/DocumentsAndMediaDocument.path

```diff
@@ -13,6 +13,11 @@
 	<td>//section[contains(@id,'DLAdminPortlet')]//span[@class='workflow-version']/span[contains(@class,'workflow-value')]</td>
 	<td></td>
 </tr>
+<tr>
```

Comment:
@Jeremy-Ch can we use any existing locator under Message.path instead of this?

Reply:
Hi @austinchiang No, we can't. I have searched from the message.path and the other files, there is not have a suitable path for it, so I added a new one.